package.json

```diff
@@ -25,6 +25,7 @@
     "test": "jest -i --coverage",
     "security_audit": "npx audit-ci --config audit-ci.json",
     "int-test": "cypress run --config video=false",
+    "int-test-single": "cypress run --config video=false --spec ",
```

Comment:
Makes it easier to run a single file, i.e.

`npm run int-test-single -- "integration_tests/integration/searchForAPrisoner.spec.ts"`